src/jade/post/excel_routines.py

```
@@ -83,14 +82,17 @@ def _add_sheet(
         if self.cfg.change_col_names:
             self._rename_columns(df, self.cfg.change_col_names)
         df.to_excel(self.writer, sheet_name=sheet_name, startrow=DF_START_ROW)
-
         # additional operation on the sheet
-        if apply_conditional and self.cfg.conditional_formatting:
+        if apply_conditional is not False and self.cfg.conditional_formatting:
+            if apply_conditional == True:
```

Ruff (0.13.1)

Avoid equality comparisons to `True`; use `apply_conditional:` for truth checks

Replace with `apply_conditional`

(E712)